<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Second table</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>First table</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="5" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QTableView" name="tableView_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
